src/redux/reducers/Dashboard/index.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-param-reassign */
+import {
+    LOADING_DASHBOARD_DATA,
+    STOP_LOADING_DASHBOARD_DATA,
+    LOAD_DASHBOARD_DATA,
+} from '../../actions/Dashboard/index';
+
+const initialState = {
+    totals: {
+        orders: 0,
+        users: 0,
+        fallowers: 0,
+        errors: 0,
+        successfulyOrdersAmount: 0,
+        totalUsers: 0,
+        fallowersActive: 0,
+    },
+    chartsData: {
+        successfulyOrders: {
+            labels: [],
+            value: [0, 0, 0, 0, 0],
+        },
+        totalUsers: {
+            labels: [],
+            value: [0, 0, 0, 0, 0],
+        },
+        fallowersActive: {
+            labels: [],
+            googleValue: [0, 0, 0, 0, 0, 0, 0],
+            facebookValue: [0, 0, 0, 0, 0, 0, 0],
+        },
+    },
+    lastMessages: [],
+    newUsers: [],
+    isLoading: false,
+};
+
+function loadData(state, action) {
+    const newState = {
+        ...state,
+        ...action.totals,
+        ...action.chartsData,
+        lastMessages: action.lastMessages,
+        newUsers: action.newUsers,
+    };
+    return newState;
+}
+
+function setIsLoading(state, action) {
+    const isLoading = {
+        isLoading: state.isLoading,
+    };
+    const newState = {
+        ...state,
+        ...isLoading,
+        isLoading: action,
+    };
+    return newState;
+}
```

Comment:
it is better if you make two functions (FUNCTIONS SHOULD DO ONE THING, THEY SHOULD DO IT WELL, THEY SHOULD DO IT ONLY):
- startLoading
- stopLoading
and you only need:
```
function startLoading(state) {
    return {
        ...state,
        isLoading: true,
    };
}
```
it is more descriptive say stopLoading() than setLoading(false)